### DISCUSSION TITLE
V大考虑搞个 OpenSingBox 吗

### DISCUSSION THREAD

**Discussion by screousi0103** [vernesong/OpenClash#4194] [Ideas]
sing-box 相比 mihomo 更适合跑在路由器上(内核修bug也快)
直接跑裸核用tun速度慢，不用tun就需要自己配防火墙规则
用了一圈没发现好用的 Sing Box 客户端(home shell neko)
希望V大适能考虑一下

**Comment by fejeIaqbhbv** [comment]
+1
现在meta的占用太高了，基本是原版内核或sb的两三倍了
小bug一堆，tun也经常出问题，莫名其妙触发，重启就恢复

**Reply by vernesong** [reply]
已经有homeproxy了，应该功能很完善了

**Reply by fejeIaqbhbv** [reply]
homeproxy更像ssrp，只能实现内核三分之二的功能效果

目前的第三方的sb客户端用起来总有别扭的地方

V大有精力的前提下希望能考虑考虑

**Reply by vernesong** [reply]
各自有各自的观点，再做一个也不会有多好用，Sing Box 的配置本身就不好写
